tests/sentry/runner/commands/test_backup.py

```diff
@@ -547,12 +549,17 @@ def test_encryption_with_gcp_kms_decryption(
         self.cli_encrypted_import_then_export_use_gcp_kms("users", fake_kms_client)
 
 
+@patch("sentry.backup.imports.ImportExportService.get_importer_for_model")
 class BadImportExportDomainErrorTests(TransactionTestCase):
-    def test_import_integrity_error_exit_code(self):
-        # First import should succeed.
-        rv = CliRunner().invoke(import_, ["global", GOOD_FILE_PATH])
-        assert rv.exit_code == 0, rv.output
-
+    def test_import_integrity_error_exit_code(self, get_importer_for_model):
```

Comment:
Since global imports now reset the database to a clean state, need to mock out this function to error out

Reply:
Thanks for doing this, good mock!